test/micro/org/openjdk/bench/java/util/UUIDBench.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2020, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
nit: other files should also have this 2021 update. It seems most of them are not updated and still uses 2020.

Reply:
fixed